apps/api/src/app/environments-v1/usecases/output-renderers/expand-email-editor-schema.usecase.ts

```diff
@@ -28,60 +33,98 @@ export class ExpandEmailEditorSchemaUsecase {
     return hydratedEmailSchema;
   }
 
-  private async processShowAndForControls(
-    variables: Record<string, unknown>,
+  private async processSpecialNodeTypes(variables: Record<string, unknown>, rootNode: TipTapNode): Promise<TipTapNode> {
```

Comment:
Something is a bit off with naming. Is the first parameter Novu Variables? Isn't `fullPayloadForRender` of [this type](https://github.com/novuhq/novu/blob/05dec8b29abfeb50a1de4ee484974b1ede4abdab/apps/api/src/app/environments-v1/usecases/output-renderers/render-command.ts#L7)?

Reply:
correct Novu Variables and fullPayloadForRender are equal.